t/t2807-dump-cmd.t

```diff
@@ -256,6 +256,47 @@ test_expect_success UTF8_LOCALE 'restore UTF-8 dump' '
 	    flux kvs get ƒuzzybunny)
 '
 
+# Cover value with a very large number of appends
+
+test_expect_success LONGTEST 'remove backing file and load content-sqlite' '
```

Comment:
I don't think the LONGTEST tests are likely to be run automatically anywhere (correct me if I'm wrong).
Could LONGTEST be dropped if the number of appends were reduced?

Reply:
nope, I just run LONGTEST when I run tests :-)

Hmmmm, I think having "quite large" is important, even if it's only when I run personally.

Perhaps we add something "medium", like 50-100 appends that can run all the time?